causallearn/utils/cit.py

```diff
@@ -58,7 +58,7 @@ def _unique(column):
             'gsq': self.gsq
         }
 
-    # TODO (@aoqi): Modify the function to support the case where the non-conditional sets are multi-dimentional.
+    # TODO: Modify the function to support the case where the non-conditional sets are multi-dimentional.
```

Comment:
Hi Aoqi, I have done this part (multi-dim unconditional sets) in #62.

Reply:
Great, thanks!